backend/src/main/java/com/votogether/domain/member/dto/MemberDetailRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.votogether.domain.member.dto;
+
+import com.votogether.domain.member.entity.Gender;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+
+@Schema(description = "회원 상세 정보 수정 요청")
+
+public record MemberDetailRequest(
```

Comment:
`P3`
```suggestion
@Schema(description = "회원 상세 정보 수정 요청")
public record MemberDetailRequest(
```
불필요한 공백이 들어가있어요 ㅎㅎ